src/core_modules/capture-core/components/Tooltips/TooltipOrgUnit/TooltipOrgUnit.component.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import React from 'react';
+import { Tooltip } from '@dhis2/ui';
+
+type Props = {
+    orgUnitName: string,
+    ancestors?: Array<string>,
+    tooltip?: string,
+};
+
+export const TooltipOrgUnit = ({ orgUnitName, ancestors = [], tooltip }: Props) => {
+    const fullPath = [...ancestors, orgUnitName].join(' / ');
+    const [before, after] = tooltip ? tooltip.split(orgUnitName) : [orgUnitName];
```

Comment:
This will not work correctly if somebody has an orgunit called `Start`... I looked a bit into ways around this issue: the one that seems like the least work to implement is to redefine `tooltip` to be an array of strings. We could then get such arrays by tampering with how we generate the translations; first we could use a delimiter symbol like `\0` in place of the orgunit name, and then we could split the resulting string with respect to the delimiter.

Reply:
Talked to Henrik. Let's simplify and avoid the splitting.
https://github.com/dhis2/capture-app/pull/3720#discussion_r1738263188